site/src/components/page-faq/question-section.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, h, Listen, Prop, State } from "@stencil/core";
+
+import { Fragment } from "../../util";
+
+/**
+* Component to take groups of faq questions and display them property
+* Also holds logic to toggle section open and closed
+*/
+@Component( {
+    tag: "question-section",
+    styleUrl: "page-faq.scss",
```

Comment:
Should probably add own CSS file here. The styles from page-faq will cascade since there is no shadow-DOM in play so only CSS specific to this component should be needed.

Reply:
sounds good, I am still pretty new to stenciljs so I am still learning how to effectively use the shadow dom. Will remove, thanks!